Sources/TracingInstrumentation/Span.swift

```diff
@@ -42,10 +42,12 @@ public protocol Span {
     mutating func addEvent(_ event: SpanEvent)
 
     /// Record an error of the given type described by the the given message.
+    ///
+    /// It's up to the individual `TracingInstrument` to handle this.
```

Comment:
no need for this comment I think? 

Since we dont provide an impl, it is known that someone has to implement and it's up to it to defined how they handle it.

Reply:
Okay, I added it because I thought some users might have an assumption about what happens here. But I agree that using a protocol communicates that to the users already anyways.

Reply:
```suggestion
```